### ISSUE TITLE
React native:  No Bugsnag API Key set, js engine: hermes

### ISSUE THREAD

**Issue by v3solutions** [bugsnag/bugsnag-js#998]
Hi,
Since upgrading bugsnag to latest version, can't make it work on adroid. Crashes on opening, however it does send a crash report to my bugsnag account. Bugsnag error: "No Bugsnag API Key set, js engine: hermes". It works no problem with ios.
Api key is set inside AndroidManifest.xml. I've tried inside App.js with and without apiKey:

```
import Bugsnag from '@bugsnag/react-native';
Bugsnag.start({
  apiKey: '90d03f98aa81640e********',
});
```

"react-native": "0.63.2"
"@bugsnag/react-native": "^7.3.1"

using with hermes.

**Comment by phillipsam** [comment]
Hi @v3solutions - are you able to share more about your reproduction case with us?

**Comment by steve-nester-uk** [comment]
@v3solutions we have been able to reproduce a similar issue, which is possibly related to the use of minifyEnabled / proguard for release builds.  We are still investigating but would be interested to know if your issue only happens during release builds (e.g. --variant=release) and whether you are using minifyEnabled?

**Comment by v3solutions** [comment]
@steve-nester-uk sorry for a delay. I'm using minifyEnabled, it does work as a debug. Seems only release build has this issue. If I comment out 
```
Bugsnag.start({
});
``` 
it works too.
Let me know if you need anything. I'll be more active this week.

**Comment by tobiasbu** [comment]
Hi, I recently upgrade React Native to 0.63.2 and Bugsnag to latest version and I started to receiving the "No Bugsnag API Key set" error in Android.

As suggested by @steve-nester-uk disabling `enableProguardInReleaseBuilds`/`minifyEnabled` option worked for me in release variant. Thanks!

**Comment by mattdyoung** [comment]
@tobiasbu - thanks for the info. Yes, we can confirm the bug can affect projects whether or not they use Hermes. We're working on the fix.

**Comment by johnkiely1** [comment]
This has now been fixed and released in [v7.3.4](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.3.4)
